mmtbx/validation/clashscore.py

```diff
@@ -9,11 +9,11 @@
 from libtbx.utils import Sorry
 from libtbx import easy_run
 import libtbx.load_env
+import libtbx.math_utils
```

Comment:
Change the import to `from libtbx.math_utils import cmp` so nothing else needs to be changed.